vulnerabilities/models.py

```diff
@@ -10,8 +10,9 @@
 import hashlib
 import json
 import logging
-import uuid
 
+from cwe import Database
+from django.contrib.postgres.fields import ArrayField
```

Comment:
Do you mind to rather use a JSONField? this is more portable than the postgres-only ArrayField

Reply:
No Problem, I will change it. but what is the best way to store CWE in the database as ( int or string ) 
{ "cwe_ids" : ['CWE-327', 'CWE-325' ]}
{ "cwe_ids" : [327 ,  325 ]}